primitives/staking/src/offence.rs

```diff
@@ -37,6 +37,29 @@ pub type Kind = [u8; 16];
 /// so that we can slash it accordingly.
 pub type OffenceCount = u32;
 
+/// In case of an offence, which conditions get an offender validator disabled
+#[derive(
+	Clone,
+	Copy,
+	PartialEq,
+	Eq,
+	Hash,
+	PartialOrd,
+	Ord,
+	Encode,
+	Decode,
+	sp_runtime::RuntimeDebug,
+	scale_info::TypeInfo,
+)]
+pub enum DisableStrategy {
+	/// Independently of slashing, this offence will not disable the offender
+	Never,
```

Comment:
For now this would be unused, but never say `Never` :P

Reply:
Yeah, this was added mostly, because it was easy to add.